test/TestCase.h

```diff
@@ -41,6 +41,8 @@ class TestCase
 		langutil::EVMVersion evmVersion;
 		std::vector<boost::filesystem::path> vmPaths;
 		bool enforceCompileViaYul;
+		bool enforceGasCost;
```

Comment:
```suggestion
		bool enforceCompileViaYul = false;
		bool enforceGasCost = false;
```

PODs should just *always* be member-initialized everywhere no matter what, unless there's very good reason not to (which rarely happens)...